pkgs/tools/audio/headsetcontrol/default.nix

```
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, hidapi
+}:
+
+stdenv.mkDerivation rec {
+  pname = "headsetcontrol";
+  version = "2.6";
+
+  src = fetchFromGitHub {
+    owner = "Sapd";
+    repo = "HeadsetControl";
+    rev = version;
+    sha256 = "0a7zimzi71416pmn6z0l1dn1c2x8p702hkd0k6da9rsznff85a88";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    hidapi
+  ];
+
+  configurePhase = ''
+    cmake -B "build" .
+  '';
+
+  buildPhase = ''
+    make -C "build"
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp build/headsetcontrol $out/bin
+
+    ${
+      if stdenv.isLinux then ''
+        mkdir -p $out/lib/udev/rules.d
+        cp build/70-headsets.rules $out/lib/udev/rules.d
+    '' else ""
+    }
```

```suggestion
    ${lib.optionalString stdenv.isLinux ''
      mkdir -p $out/lib/udev/rules.d
      cp build/70-headsets.rules $out/lib/udev/rules.d
    ''}
```
(Note: this may not be relevant, if we can drop the phase override)